Functions.Templates/Templates/EventGridBlobTrigger-JavaScript/function.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`Functions.Templates/Templates/EventGridBlobTrigger-JavaScript` is missing metadata.json file

Reply:
Added